code/game/machinery/machinery.dm

```diff
@@ -157,10 +157,11 @@ Class Procs:
 		PoolOrNew(/obj/effect/overlay/temp/emp, loc)
 	..()
 
-/obj/machinery/proc/open_machine()
+/obj/machinery/proc/open_machine(var/drop = 1)
```

Comment:
Ok, so when exactly DO you need to specify var/ ?
Because I've had times when it wouldn't compile if I didn't do that.